packages/amplify_api/lib/amplify_api.dart

```diff
@@ -43,21 +43,21 @@ class AmplifyAPI extends APIPluginInterface {
 
   // ====== GraphQL =======
   @override
-  GraphQLOperation<T> query<T>({@required GraphQLRequest<T> request}) {
```

Comment:
Same as the other NS PRs, `import 'package:flutter/foundation.dart';` is now used with `@required` removed.

Reply:
removed unnecessary import